autoware_adapi_v1_msgs/vehicle/msg/VehicleMotion.msg

```diff
@@ -0,0 +1,5 @@
+std_msgs/Header header
+float64 velocity
+float64 angular_velocity
+float64 acceleration
+float64 steering
```

Comment:
> Some vehicles do not have steering. How about using the more generic types Twist and Accel in geometry_msgs?
Ya, I think this might be a better approach, let me redesign

> Also, what about covariance?
For autoware report tpic, they do not include covariance, if we just publishing the vehicle current status, I do not think we need to have covariance. Hope do you think?
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_vehicle_msgs/msg/VelocityReport.idl
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_vehicle_msgs/msg/VehicleOdometry.idl
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_vehicle_msgs/msg/SteeringReport.idl

Reply:
I think for the steering naming, we should change to tire angle to prevent confuse.

Reply:
I had a misunderstanding. Actually steering is the tire angle. So what we have to decide is whether to use the tire angle or the vehicle angle.

Reply:
> For autoware report tpic, they do not include covariance, if we just publishing the vehicle current status, I do not think we need to have covariance. How do you think?

I agree. I mistook it for a comment on pose.

Reply:
> I had a misunderstanding. Actually steering is the tire angle. So what we have to decide is whether to use the tire angle or the vehicle angle.

For the vehicle angle, do you the angle of the whole vehicle, which come from the AckermannLateralCommand?

Reply:
I found that each of pose and twist and accel will have covariance In localization. 
https://github.com/orgs/autowarefoundation/discussions/2455#discussioncomment-2241651

Reply:
I think we will need to decide whether the twist and accel will be get from localization or vehicle itself.

Reply:
I heard that using localization because vehicle may be less accurate.

Reply:
I see, okay sure, then I will change to use localization

Reply:
changed at 
ad1c1ef